### ISSUE TITLE
module 'matplotlib.cbook' has no attribute 'is_numlike'

### ISSUE THREAD

**Issue by brianpenghe** [scverse/scanpy#1227]
I was running Scanpy 1.4.5.1 on Jupyter Notebook
My matplotlib version is 3.1.3

I ran paga using these commands:
```
sc.tl.paga(bdata,groups='leiden')
sc.pl.paga(bdata, plot=False)
sc.pl.paga(bdata)
````

The third command gave me this error:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-65-5027c99fe1bd> in <module>
----> 1 sc.pl.paga(bdata)

/usr/local/lib/python3.6/dist-packages/scanpy/plotting/_tools/paga.py in paga(adata, threshold, color, layout, layout_kwds, init_pos, root, labels, single_component, solid_edges, dashed_edges, transitions, fontsize, fontweight, fontoutline, text_kwds, node_size_scale, node_size_power, edge_width_scale, min_edge_width, max_edge_width, arrowsize, title, left_margin, random_state, pos, normalize_to_color, cmap, cax, colorbar, cb_kwds, frameon, add_pos, export_to_gexf, use_raw, colors, groups, plot, show, save, ax)
    541                 single_component=single_component,
    542                 arrowsize=arrowsize,
--> 543                 pos=pos,
    544             )
    545             if colorbars[icolor]:

/usr/local/lib/python3.6/dist-packages/scanpy/plotting/_tools/paga.py in _paga_graph(adata, ax, solid_edges, dashed_edges, adjacency_solid, adjacency_dashed, transitions, threshold, root, colors, labels, fontsize, fontweight, fontoutline, text_kwds, node_size_scale, node_size_power, edge_width_scale, normalize_to_color, title, pos, cmap, frameon, min_edge_width, max_edge_width, export_to_gexf, colorbar, use_raw, cb_kwds, single_component, arrowsize)
    756         with warnings.catch_warnings():
    757             warnings.simplefilter("ignore")
--> 758             nx.draw_networkx_edges(nx_g_solid, pos, ax=ax, width=widths, edge_color='black')
    759     # draw directed edges
    760     else:

/usr/local/lib/python3.6/dist-packages/networkx/drawing/nx_pylab.py in draw_networkx_edges(G, pos, edgelist, width, edge_color, style, alpha, arrowstyle, arrowsize, edge_cmap, edge_vmin, edge_vmax, ax, arrows, label, node_size, nodelist, node_shape, **kwds)
    609         # value globally, since the user can instead provide per-edge alphas
    610         # now.  Only set it globally if provided as a scalar.
--> 611         if cb.is_numlike(alpha):
    612             edge_collection.set_alpha(alpha)
    613 

AttributeError: module 'matplotlib.cbook' has no attribute 'is_numlike'

```

I've been searching online and found some related threads like [this](https://github.com/palash1992/GEM/issues/51) and [this](https://stackoverflow.com/questions/53421905/matplotlib-attributeerror-module-matplotlib-cbook-has-no-attribute-define-a)

Is there a solution that doesn't require me to downgrade my matplotlib?

**Comment by Chenyanjuan1993** [comment]
I meet the same problem.

You could use:

pip install matplotlib==2.2.3

I tried just. And it done.

**Comment by flying-sheep** [comment]
We require matplotlib 3.x for other parts of scanpy, so that’s not a real solution. As you can see in the traceback, the error happens in `networkx`. It has been fixed in networkx/networkx#3179 ([networkx 2.3](https://networkx.github.io/documentation/stable/release/release_2.3.html)) in April 2019. So you should upgrade networkx instead of downgrading matplotlib.
